### PR TITLE
Added new component for guardians of the galaxy - with endpoint guard…

### PR DESCRIPTION
**Pull request by wbylbie**
Guardians homework

### CODE REVIEW

File: Practice/src/main/java/com/example/learn/practice/guardians.java [hunk 1]
```diff
@@ -0,0 +1,18 @@
+package com.example.learn.practice;
+
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/guardians")
+public class guardians {
```
Comment (softwarewrestler) [inline Practice/src/main/java/com/example/learn/practice/guardians.java]: By convention classnames are uppercase.
This class should be called Guardians

File: Practice/src/main/java/com/example/learn/practice/guardians.java [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+	@GetMapping("/members")
+	    public String getMembers() {
+	    return "Star-Lord, Gamora, Drax, Rockett, Groot";
```
Comment (softwarewrestler) [inline Practice/src/main/java/com/example/learn/practice/guardians.java]: What about Nebula, Mantis and Kraglin?

**Review by softwarewrestler — CHANGES_REQUESTED** [review]

**Comment by wbylbie** [comment]
Updated changes

File: Practice/src/main/java/com/example/learn/practice/Guardians.java [hunk 3]
```diff
@@ -5,12 +5,12 @@
 import org.springframework.web.bind.annotation.RestController;
 
 @RestController
-@RequestMapping("/guardians")
+@RequestMapping("/Guardians")
```
Comment (softwarewrestler) [inline Practice/src/main/java/com/example/learn/practice/Guardians.java]: This is part of the endpoint mapping, it should be lower case.

File: Practice/src/main/java/com/example/learn/practice/Guardians.java [hunk 4]
```diff
@@ -5,12 +5,12 @@
...
 public class guardians {
```
Comment (softwarewrestler) [inline Practice/src/main/java/com/example/learn/practice/Guardians.java]: This is the class name.
This is the one that should be uppercase.

**Review by softwarewrestler — CHANGES_REQUESTED** [review]

**Pull request merged**